umake/frameworks/android.py

```diff
@@ -76,12 +76,12 @@ def parse_download_link(self, tag, line, in_download):
 
 class AndroidStudio(umake.frameworks.baseinstaller.BaseInstaller):
 
-    def __init__(self, category):
+    def __init__(self, category, force_loading):
```

Comment:
I modified the list argument for each framework, this way is better than the previous for the future if we need to add a new mandatory argument.

It remains to modify all the tests frameworks, I will do it soon ;)